include/ostro-xt-supported-recipes.txt

```diff
@@ -0,0 +1,3 @@
+libevent@core
+libxscrnsaver@core
+nss@core
```

Comment:
Contents in this file depend highly on what's included in the default Ostro OS XT image and user configuration in local.conf. How are you going to maintain the list to keep it up-to-date and valid for common configurations?